ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -351,7 +355,12 @@ Create DSCInitialization CustomResource Using Test Variables
     Run    sed -i'' -e 's/<dsci_name>/${dsci_name}/' ${file_path}dsci_apply.yml
     Run    sed -i'' -e 's/<application_namespace>/${APPLICATIONS_NAMESPACE}/' ${file_path}dsci_apply.yml
     Run    sed -i'' -e 's/<monitoring_namespace>/${MONITORING_NAMESPACE}/' ${file_path}dsci_apply.yml
-    Run    sed -i'' -e 's/<monitoring_value>/Managed/' ${file_path}dsci_apply.yml
+    ${ENABLE_OBSERVABILITY_STACK} =    Get Variable Value    ${ENABLE_OBSERVABILITY_STACK}    true
+        IF    "${ENABLE_OBSERVABILITY_STACK}" == "true"
+            Run    sed -i'' -e 's/<monitoring_value>/Managed/' ${file_path}dsci_apply.yml
+        ELSE
+            Run    sed -i'' -e 's/<monitoring_value>/Removed/' ${file_path}dsci_apply.yml
```

Comment:
This will mean we will be unable to test the current monitoring stack (limited to just managed clusters), right? Because it will be set to Removed.
I think until we are fully onboarded on the new stack, we should keep this part as managed always. And new stack will be spun up just in case we patch the Monitoring config with newly-added configuration (metrics, traces keys).